docker/spicedb/schema.zed

```diff
@@ -361,7 +361,8 @@ definition domain {
 	permission group_add_role_users_permission = group_add_role_users + team->group_add_role_users + organization->admin
 	permission group_remove_role_users_permission = group_remove_role_users + team->group_remove_role_users + organization->admin
 	permission group_view_role_users_permission = group_view_role_users + team->group_view_role_users + organization->admin
-
+	
+	permission send_invitation = admin
```

Comment:
Why don't we reuse the permission `admin` or `add_role_users_permission`?, 
https://github.com/absmach/supermq/blob/d842182d160a442e943ed97a07902f54512275da/docker/spicedb/schema.zed#L323
Because , invitation is similar to adding user to a role.

Reply:
adding user to role is not exactly the same. since invitations can be rejected.

The idea here is with billing if we are to limit the number of users in a workspace. we need to limit sending invitations and accepting invitations at the same time. 

some psudocode:

1. Send user inviation
2. Billing callback checks if domain-members limit is exceeded. if not invitation sending is authorized

1. when user is accepting invitation ("adding role user")
2. billing will also check domain members limit. Allow the user to be added to domain or not

Reply:
Got it, 
Then Can we have separate relation like `relation send_invitations: role#member`? 
and separate permission like `permission send_invitation_permission = send_invitations + organization->admin`

What do you think about this approach  ?

Reply:
Not sure if a new relation is needed, since it will add the number of options the user has on actions, but I don't mind if you'd like to add it on mg. Billing only cares about the permission in the auth request

Reply:
@SammyOina 
This can be avoided , if we move callout auth of auth service 
I have created issue here for this 
https://github.com/absmach/supermq/issues/2804